OpenRA.Mods.Common/Traits/RepairsUnits.cs

```diff
@@ -23,6 +24,9 @@ public class RepairsUnitsInfo : PausableConditionalTraitInfo
 		[Desc("Time (in ticks) between two repair steps.")]
 		public readonly int Interval = 24;
 
+		[Desc("Damage type used for the repair.")]
+		public readonly BitSet<DamageType> RepairDamageType = default(BitSet<DamageType>);
```

Comment:
RepairDamageTypes would be better as this can take mutiple values.

Reply:
Good idea.